pkg/expect/expect_test.go

```diff
@@ -72,7 +72,7 @@ func TestExpectFuncTimeout(t *testing.T) {
 	}
 
 	err = ep.Close()
-	require.ErrorContains(t, err, "unexpected exit code [143] after running [/usr/bin/tail -f /dev/null]")
+	require.ErrorContains(t, err, "unexpected exit code [143]")
```

Comment:
minor comment:
```suggestion
	require.Regexp(t, "unexpected exit code .*143.* after running.*tail", err.Error())
```

Reply:
wondering if the current format is easier to understand and reason about? the pattern is used elsewhere too:
```
tests/e2e/ctl_v3_auth_test.go:                  require.ErrorContains(cx.t, err, "unexpected exit code")
tests/e2e/ctl_v3_elect_test.go:         require.ErrorContains(cx.t, err, "unexpected exit code")
tests/e2e/ctl_v3_lock_test.go:          require.ErrorContains(cx.t, err, "unexpected exit code")
tests/e2e/v3_curl_test.go:      require.ErrorContains(cx.t, err, "unexpected exit code")
```

But happy to change if you feel strongly about it.

Reply:
It isn't a big deal, feel free to keep it as it's.